src/test/java/Web/TestCases/NewDigitalId/ExpectedElementsArePresentAtStepIdType.java

```diff
@@ -1,65 +1,49 @@
+import Pages.App.*;
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.NewDigitalIdRequestPage;
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.Steper.IdType.IdTypeStepPage;
+import Pages.Services.InternalServices.InternalServicesPage;
+import Pages.Services.ServicesPage;
 import Web.TestCases.Login.NeomUserCanBeSuccessfullyLoggedIn;
-import Web.SiteTree.Services.*;
-import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.ElementsCollection;
-import org.testng.annotations.Test;
-
-import java.lang.reflect.Array;
-import java.time.Duration;
+import org.junit.jupiter.api.BeforeEach;
+import org.testng.annotations.*;
 import java.util.Arrays;
 import java.util.List;
-
 import static com.codeborne.selenide.Condition.*;
 import static com.codeborne.selenide.Selenide.*;
 
 public class ExpectedElementsArePresentAtStepIdType extends NeomUserCanBeSuccessfullyLoggedIn {
-    @Test
-    public void expectedElementsArePresentAtStepIdType(){
+    private App app;
+    @BeforeMethod
+    public void setUp(){
+        app = new App();
+    }
+    @BeforeEach
+    public void login() {
         loginFlow();
+    };
 
-        Services services = new Services();
-        InternalServices internalServices = new InternalServices();
-        NewDigitalIdRequest newDigitalIdRequest = new NewDigitalIdRequest();
-        IdTypeStep idTypeStep = new IdTypeStep();
-
-        services.servicesButton.click();
-        internalServices.digitalIdRequestsButton.click();
-        newDigitalIdRequest.newDigitalIdRequestsButton.click();
-
-        idTypeStep.stepNaming.shouldHave(text("ID type"));
-        idTypeStep.titleNaming.shouldHave(text("Select ID type"));
-        idTypeStep.subTitleNaming.shouldHave(text("Please select desired ID type from the list. Your choice will be applied to all people for whom you are requesting ID."));
-        idTypeStep.idTypeDropdown.shouldBe(visible);
-        //idTypeStep.idTypeDropdown.shouldBe(Condition.visible, Duration.ofSeconds(10));
-        idTypeStep.backButton.shouldBe(visible, enabled);
-        idTypeStep.nextButton.shouldBe(visible, disabled);
+    @Test
+    public void expectedElementsArePresentAtStepIdType(){
+        app.services.servicesButton.click();
+        app.internalServices.digitalIdRequestsButton.click();
+        app.newDigitalIdRequest.newDigitalIdRequestsButton.click();
+        app.idTypeStep.stepNaming.shouldHave(text("ID type"));
+        app.idTypeStep.titleNaming.shouldHave(text("Select ID type"));
+        app.idTypeStep.subTitleNaming.shouldHave(text("Please select desired ID type from the list. Your choice will be applied to all people for whom you are requesting ID."));
+        app.idTypeStep.idTypeDropdown.shouldBe(enabled);
+        app.idTypeStep.backButton.shouldBe(enabled);
+        app.idTypeStep.nextButton.shouldBe(disabled);
     }
     @Test
     public void expectedValuesArePresentAtIdTypeDropdown(){
```

Comment:
I don't see a test logic where the stepper form is filled in, user clicks Next and appears on the next stepper. And this test has to be the critical one, as it's about logic